SQL/New_patches/2018_08_14_csrf_tokens.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE `csrf_tokens` (
+  `CsrfTokenID` int(11) NOT NULL AUTO_INCREMENT,
+  `Token` varchar(64) NOT NULL,
+  `CreatedAt` date NOT NULL,
+  `PhpSessionID` varchar(30) NOT NULL,
+  PRIMARY KEY (`CsrfTokenID`)
+) ENGINE=InnoDB AUTO_INCREMENT=24 DEFAULT CHARSET=latin1;
```

Comment:
suggest a primary number (at least an odd one) for auto_increment. This way, it wont try to reuse an already used number after circling around.


Any method to remove the old (>2 days) token from the table? In the long run, it will use lots of space.

Reply:
@PapillonMcGill Makes sense. I just dumped the table schema to create this. Don't know where 24 came from.

As for deleting the old tokens, I'm open to suggestions. Maybe a cron script to remove old cookies?

Reply:
@johnsaigle I also suggest a cron job to remove all cookies older that 48 hour. This would keep the size of the table low enough and should not have impact (extremely low) on CPU time.